components/StandardListItem/StandardListItem.jsx

```diff
@@ -0,0 +1,46 @@
+import {PropTypes, Component } from 'react'
+import React from 'react'
+
+require('./StandardListItemStyles.scss')
+ // showIcon: true -> render the icon
+ // showLabel: true -> render the label
+ // imgSrc: source for the icon
+ // labelText: The text for the label
+ // placeIcon: defines the position of the icon. Either: top | left | right.  Default to top
+
+class StandardListItem extends Component {
```

Comment:
@nlitwin it was my bad to ask for converting this component to the full fledged ES6 style. I didn't realize that this component is not using any state. I think @k9101 originally followed the pattern which is suggested for stateless components. @k9101 sorry for going back and forth  and it would be great if you can incorporate the suggested pattern again.